DXMainClient/PreStartup.cs

```diff
@@ -77,7 +97,20 @@ public static void Initialize(StartupParams parameters)
             // Try to load translations
             try
             {
-                var translation = TranslationTable.LoadFromIniFile(ClientConfiguration.Instance.TranslationIniName);
+                TranslationTable translation;
+                var iniFileInfo = SafePath.GetFile(ProgramConstants.GamePath, ClientConfiguration.Instance.TranslationIniName);
+
+                if (iniFileInfo.Exists)
+                {
+                    translation = TranslationTable.LoadFromIniFile(iniFileInfo.FullName);
+                }
+                else
+                {
+                    Logger.Log("Failed to load the translation file. File does not exist.");
+
+                    translation = new TranslationTable();
+                }
```

Comment:
This code looks really generic with what I've seen previously, maybe should somehow consolidate it?